packages/ui-color-utils/src/conversions.ts

```diff
@@ -91,10 +91,15 @@ const colorToHsla = (
 }
 
 export {
-  color2hex,
-  colorTohex8,
+  colorToHex,
+  colorToHex8,
   colorToHsva,
   colorToHsla,
   colorToRGB,
-  colorToRGB as hexToRgb
+  //these are needed for backward compatibility
+  //TODO deprecate
+  colorToRGB as hexToRgb,
```

Comment:
As far as I can tell, the `hexToRGB` is only used in the new component, so it might be enough to refactor there. Only the `color2hex` util existed here before.

Reply:
Yeah, very true, nice catch